[DM-2442] [changelog] opcua device type encoding bugfix (#2479)
SoftwareAG/c8y-docs: content/change-logs/device-management/opcua-10-20-88-0-fix-device-type-encoding.md

---
date: 

> title: Fixed special characters encoding in OPC-UA device types

Reviewer: ```suggestion
title: Special character encoding in OPC-UA device types is now available
```

> title: Special character encoding in OPC-UA device types is now available

Reviewer 2: ```suggestion
title: Special character encoding in OPC UA device types is now available
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-Tf05_KQ-B
    label: OPC UA
build_artifact:
  - value: tc-MLn0oFRX-
    label: opcua

> ticket: DM-24422

Reviewer 3: ```suggestion
ticket: DM-2442
```

> version: 10.20.88.0

Reviewer 3: The change is not merged yet, don't forget to update here if the actual version differs

---

> When OPC-UA device type was created with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This problem is now fixed - gateway encodes and decodes special characters as expected.  

Reviewer: ```suggestion
Previously, if you created an OPC-UA device type with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This problem is now fixed, the gateway encodes and decodes special characters as expected.  
```

> Previously, if you created an OPC-UA device type with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This problem is now fixed, the gateway encodes and decodes special characters as expected.  

Reviewer 2: ```suggestion
Previously, when creating an OPC UA device type with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This issue is now fixed, the gateway encodes and decodes special characters as expected.  
```
